### DISCUSSION TITLE
Suggestion: RE of infamous MODCONV tool for modern platforms

### DISCUSSION THREAD

**Discussion by ElvargThemSelf** [grumpycoders/pcsx-redux#926] [Ideas]
When you're going to RE the MODCONV since currently there's no way to convert .mod files to .hit atm?

**Comment by nicolasnoble** [comment]
It's fairly low on my list, primarily because the old tool works fine within DOSBOX. You can absolutely just spin up dosbox and run MODCONV.EXE in it on any platform. But yes, I'll write a tool at some point.

On a side note, the tool doesn't really need to be reversed. The file format is fairly straightforward. A .hit file is just a .mod file with its magic changed from MOD to HIT, and the instruments converted to adpcm.
